opensrp-chw/src/main/java/org/smartregister/chw/activity/ReferralFollowupActivity.java

```diff
@@ -41,6 +42,8 @@ public static void startReferralFollowupActivity(Activity activity, String taskI
         intent.putExtra(Constants.ACTIVITY_PAYLOAD.REFERRAL_FOLLOWUP_FORM_NAME, getReferralFollowupForm(referralType));
         intent.putExtra(Constants.ACTIVITY_PAYLOAD.JSON_FORM, getReferralFollowupForm(referralType));
         intent.putExtra(TASK_IDENTIFIER, taskIdentifier);
+        if (activity.getClass() == AtReferralDetailsViewActivity.class)
+            intent.putExtra("IS_COMING_FROM_REFERRAL_DETAILS", true);
```

Comment:
consider putting the "IS_COMING_FROM_REFERRAL_DETAILS" string in constant

Reply:
Thanks @Andati - Makes sense, I will add it as a constant